### ISSUE TITLE
Podman v3 can't pull any image because of default policy

### ISSUE THREAD

**Issue by Friz-zy** [containers/podman#9257]
/kind bug

**Description**

Podman show an error while trying pull any image from any public registry like docker.io or registry.fedoraproject.org

**Steps to reproduce the issue:**

```
# podman pull fedora
Resolved short name "fedora" to a recorded short-name alias (origin: /etc/containers/registries.conf.d/shortnames.conf)
Trying to pull registry.fedoraproject.org/fedora:latest...
  Running image docker://registry.fedoraproject.org/fedora:latest is rejected by policy.
Error: Source image rejected: Running image docker://registry.fedoraproject.org/fedora:latest is rejected by policy.
```

**Output of `podman version`:**

```
podman version 3.0.0-rc1
```

**Output of `podman info --debug`:**

```
host:
  arch: amd64
  buildahVersion: 1.19.2
  cgroupManager: systemd
  cgroupVersion: v1
  conmon:
    package: 'conmon: /usr/libexec/podman/conmon'
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.0.24, commit: '
  cpus: 4
  distribution:
    distribution: ubuntu
    version: "18.04"
  eventLogger: journald
  hostname: zen
  idMappings:
    gidmap: null
    uidmap: null
  kernel: 5.4.0-54-generic
  linkmode: dynamic
  memFree: 9947365376
  memTotal: 25099407360
  ociRuntime:
    name: crun
    package: 'crun: /usr/bin/crun'
    path: /usr/bin/crun
    version: |-
      crun version 0.17.5-86d6-dirty
      commit: fd582c529489c0738e7039cbc036781d1d039014
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  os: linux
  remoteSocket:
    exists: true
    path: /run/podman/podman.sock
  security:
    apparmorEnabled: true
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: false
    seccompEnabled: true
    selinuxEnabled: false
  slirp4netns:
    executable: ""
    package: ""
    version: ""
  swapFree: 5578420224
  swapTotal: 5578420224
  uptime: 74h 17m 34.53s (Approximately 3.08 days)
registries: {}
store:
  configFile: /etc/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions:
    overlay.mountopt: nodev,metacopy=on
  graphRoot: /var/lib/containers/storage
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "true"
  imageStore:
    number: 34
  runRoot: /var/run/containers/storage
  volumePath: /var/lib/containers/storage/volumes
version:
  APIVersion: 3.0.0
  Built: 0
  BuiltTime: Thu Jan  1 03:00:00 1970
  GitCommit: ""
  GoVersion: go1.15.2
  OsArch: linux/amd64
  Version: 3.0.0-rc1
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
ii  containernetworking-plugins   0.9.0~3
ii  containers-common                1:1~4
ii  podman                                   3.0.0~0.4.rc1
ii  podman-plugins                       1.1.1~1
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide?**

Yes

**Comment by Friz-zy** [comment]
Also it's affect even layers copying between builds:
```
Error: error copying image "78b2869b282bf2f28a5e873d6ade079e83d77765223c7bcd180b77cbc8fe4751": Source image rejected: Running image containers-storage:[overlay@/var/lib/containers/storage+/var/run/containers/storage:overlay.mountopt=nodev,metacopy=on]@78b2869b282bf2f28a5e873d6ade079e83d77765223c7bcd180b77cbc8fe4751 is rejected by policy.
```

In podman v2 `/etc/containers/policy.json` just contain something like
```
    "default": [
        {
            "type": "insecureAcceptAnything"
        }
```
if I right remember, so I did it for v3 too as a hotfix

**Comment by mheon** [comment]
@vrothberg PTAL

**Comment by vrothberg** [comment]
This looks like a packaging issue. @lsm5 PTAL.

I saw the same issue on IRC last week but I couldn't reproduce.

**Comment by rhatdan** [comment]
That file is shipped as part of containers-common,
 rpm -qVf /etc/containers/policy.json

**Comment by lsm5** [comment]
Should be fixed with v3.0.0. Please reopen if issue persists.
